integrationtest/backend/tck/src/main/java/org/hibernate/search/integrationtest/backend/tck/search/projection/DistanceProjectionTypeCheckingAndConversionIT.java

```diff
@@ -74,7 +76,8 @@ public class DistanceProjectionTypeCheckingAndConversionIT {
 	@BeforeClass
 	public static void setup() {
 		setupHelper.start()
-				.withIndexes( mainIndex, compatibleIndex, rawFieldCompatibleIndex, missingFieldIndex, incompatibleIndex )
+				.withIndexes(
+						mainIndex, compatibleIndex, rawFieldCompatibleIndex, missingFieldIndex, incompatibleIndex )
```

Comment:
I see you're using our IDE code style templates; that's good, but in the future please make sure to only apply auto-formatting to portions of files you changed: that way you won't change irrelevant code in your commits.

Reply:
yes, sure! I remember it took a while to make Idea code style to match the Eclipse one back in the days while working with HV. I've cleaned this formatting up as if it never happened 🙈

Reply:
Thanks.

FWIW we now publish IDEA code styles as well: https://hibernate.org/community/contribute/intellij-idea/#code-style